### PR TITLE
Fix reshape error for onnx

### PR DESCRIPTION
**Pull request by gaurides**

### CODE REVIEW

File: .ci/jenkins/jenkins-trigger.groovy [hunk 1]
```diff
@@ -24,7 +24,7 @@
 // Parameters which ngraph-unittest uses:
 String  PR_URL = CHANGE_URL
 String  PR_COMMIT_AUTHOR = CHANGE_AUTHOR
-String  JENKINS_BRANCH = "ngraph-core-r0.22.0.rc1"
+String  JENKINS_BRANCH = "aslepko/r0.22"
```
Comment (jbobba) [inline .ci/jenkins/jenkins-trigger.groovy]: seems unrelated

Comment (gaurides) [reply .ci/jenkins/jenkins-trigger.groovy]: @aslepko recommended this change to fix the CI error 
`our old docker hub is no longer online so the openmpi test there fails`

**Comment by gaurides** [comment]
> Unfortunately some ONNX unit tests are failing with this change.

@postrational : are you referrring to the test failures in onnx-ci ?

**Comment by diyessi** [comment]
ONNX tests pass on master.
